src/main_chart.py

```diff
@@ -93,45 +109,40 @@ def get_main_chart_figure(
 ) -> plotly.graph_objs.Figure:
     # Define the AMMs and their color mappings
     amms = src.swap_amm.AMMS
-    color_map = {
-        "10kSwap_debt_token_supply": "#4C6DD0",
-        "MySwap_debt_token_supply": "#4CA7D0",
-        "SithSwap_debt_token_supply": "#4C99D0",
-        "JediSwap_debt_token_supply": "#4C83D0",
-    }
 
-    # TODO: Align colors with the rest of the app.
-    color_map_protocol = {
-        "liquidable_debt_at_interval_zkLend": "#fff7bc",  # light yellow
-        "liquidable_debt_at_interval_Nostra Alpha": "#fec44f",  # yellow
-        "liquidable_debt_at_interval_Nostra Mainnet": "#d95f0e",
-    }  # mustard yellow
-    color_map_liquidity = {"debt_token_supply": "#1f77b4"}  # blue
+    color_map = COLOR_MAP_AMM
+    color_map_protocol = COLOR_MAP_PROTOCOL
```

Comment:
why do you put constant into variable? 
It's just extra unuseful code which make code more complex, use just COLOR_MAP_PROTOCOL, COLOR_MAP_AMM, COLOR_MAP_LIQUIDITY without setting new variable